tests/test_textloader.py

```diff
@@ -59,18 +61,39 @@ def test__read_text_txt(mock_read_text):
     assert_frame_equal(expected_result.compute(), actual_result.compute().reset_index(drop=True))
 
 
-@patch("dask.dataframe.read_json")
-def test__read_text_json(mock_read_json):
+@patch("nlpretext._utils.pandasloader.read_text")
+def test__read_text_txt_pandas(mock_read):
+    # Given
+    files_path = "some_path/to_read.txt"
+    file_format = "txt"
+    encoding = "utf-8"
+    text_column = "text"
+
+    dummy_instance = TextLoader(
+        file_format=file_format,
+        use_dask=False,
+        encoding=encoding,
+        text_column=text_column,
+    )
+    dummy_instance._read_text_txt(files_path)
+
+    # Then
+    mock_read.assert_called_once_with(files_path, encoding=encoding)
```

Comment:
Result of the function is not tested, is it on purpose?

Reply:
Nope, fixing it